controllers/configmap_controller.go

```diff
@@ -141,6 +141,15 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context,
 
 	// at this point configMap will be set properly
 	if req.NamespacedName.Name == servicescm.ServicesConfigMapName {
+		// If a ConfigMap with invalid values is found, WMCO will delete and recreate it with proper values
+		if _, err := servicescm.Parse(configMap.Data); err != nil {
+			// Deleting will trigger an event for the configmap_controller, which will re-create a proper ConfigMap
+			if err = r.client.Delete(ctx, configMap); err != nil {
+				return ctrl.Result{}, err
+			}
+			r.log.V(1).Info("Deleted", "ConfigMap", kubeTypes.NamespacedName{Namespace: req.Namespace, Name: req.Name})
```

Comment:
This might actually be a useful log. I suggest making this just `Info`. But in addition to just saying `Deleted` please say why.